lib/Backend/Func.h

```diff
@@ -52,6 +52,36 @@ struct Cloner
     bool clonedInstrGetOrigArgSlotSym;
 };
 
+/*
+* This class keeps track of various information required for Stack Arguments optimization with formals.
+*/
+class StackArgWithFormalsTracker
+{
+private:
+    BVSparse<JitArenaAllocator> *  formalsArraySyms;    //Tracks Formal parameter Array - Is this Bv required explicitly?
+    StackSym**                     formalsIndexToSymIdMap; //Tracks the stack sym for each formal
```

Comment:
This is more of a formals index to sym (instead of sym id) map, right? Consider renaming it as such.